src/scripting_api/util.js

```diff
@@ -409,10 +409,17 @@ class Util extends PDFObject {
     const number = /\d+/g;
     let i = 0;
     let array;
-    const date = new Date(0);
+    const data = {
+      year: 2000,
```

Comment:
How was this value chosen, or does it the exact value not matter?

Reply:
I just checked in Acrobat and the current year is used, so let's do the same.